Mail2Bug/Email/AckEmailHandler.cs

```diff
@@ -28,21 +30,26 @@ public void SendAckEmail(IIncomingEmailMessage originalMessage, string workItemI
             var ewsMessage = originalMessage as EWSIncomingMessage;
             if (ewsMessage != null)
             {
-                HandleEWSMessage(ewsMessage, workItemId);
+                HandleEWSMessage(ewsMessage, workItem);
             }
         }
 
-        private void HandleEWSMessage(EWSIncomingMessage originalMessage, string workItemId)
+        private void HandleEWSMessage(EWSIncomingMessage originalMessage, SimpleWorkItem workItem)
         {
-            originalMessage.Reply(GetReplyContents(workItemId), _config.EmailSettings.AckEmailsRecipientsAll);
+            originalMessage.Reply(GetReplyContents(workItem), _config.EmailSettings.AckEmailsRecipientsAll);
         }
 
-        private string GetReplyContents(string workItemId)
+        private string GetReplyContents(SimpleWorkItem workItem)
         {
             var bodyBuilder = new StringBuilder();
             bodyBuilder.Append(_config.EmailSettings.GetReplyTemplate());
-            bodyBuilder.Replace("[BUGID]", workItemId);
```

Comment:
Done. I have also renamed placeholders for first-class members to [_ID], [_TITLE], [_OWNER] and [_STATE]. [BUGID] will continue to be replaced for backward compat.